src/app/pages/app-bar/app-bar.component.ts

```diff
@@ -4,52 +4,48 @@ import { map } from 'rxjs/operators';
 import { ScrollDispatcher, CdkScrollable } from '@angular/cdk/scrolling';
 
 @Component({
-  selector: 'app-app-bar',
-  templateUrl: './app-bar.component.html',
-  styleUrls: ['./app-bar.component.scss']
+    selector: 'app-app-bar',
+    templateUrl: './app-bar.component.html',
```

Comment:
should this be called 'collapsible-app-bar'?

Reply:
Removed the app bar.